<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>701</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>员工编号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>姓名</string>
       </property>
      </item>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>411</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>工资</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>部门</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查询员工信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查询工资信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查询部门信息</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>你好</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
